arwen/elrondapi/elrondei.go

```diff
@@ -2639,6 +2739,121 @@ func v1_3_createContract(
 	return 0
 }
 
+//export v1_3_deployFromSourceContract
+func v1_3_deployFromSourceContract(
+	context unsafe.Pointer,
+	gasLimit int64,
+	valueOffset int32,
+	sourceContractAddressOffset int32,
+	resultAddressOffset int32,
+	numArguments int32,
+	argumentsLengthOffset int32,
+	dataOffset int32,
+) int32 {
```

Comment:
Must also take `codeMetadata` as parameter, regardless of what the source contract has.